css/amp.css

```diff
@@ -334,3 +334,13 @@ amp-analytics {
   overflow: hidden !important;
   visibility: hidden;
 }
+
+amp-user-notification {
+  position: fixed !important;
```

Comment:
thanks @cramforce 

linking https://github.com/ampproject/amphtml/issues/1215 for reference